src/default-controls.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+
+const defaultButtonStyles = disabled => ({
+  border: 0,
+  background: 'rgba(0,0,0,0.4)',
+  color: 'white',
+  padding: 10,
+  outline: 0,
+  opacity: disabled ? 0.3 : 1,
+  cursor: 'pointer'
+});
+
+export class PreviousButton extends React.Component {
+  handleClick = e => {
```

Comment:
GLOBAL: Is there any chance we can get rid of all member-as-arrow functions? The stuff babel produces is (1) not in the inheritance chain, (2) can't be introspected (because not on prototype), and (3) is stage-1.

Actually, could we maybe remove all of:

```
    "@babel/plugin-proposal-class-properties",
    "@babel/plugin-proposal-object-rest-spread",
    "@babel/plugin-proposal-optional-chaining"
```

and stick with env + react?